core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportTabDelimData.java

```diff
@@ -420,6 +423,7 @@ public void testImportRppaData() throws Exception {
         geneticProfile.setDatatype("LOG2-VALUE");
         geneticProfile.setProfileName("RPPA Data");
         geneticProfile.setProfileDescription("RPPA Data");
+        geneticProfile.setReferenceGenomeId(1);
```

Comment:
and here

Reply:
reference_id assigned not auto generated. will not change.